.github/workflows/command-compile.yml

```
@@ -26,6 +26,12 @@ jobs:
       base_ref: ${{ steps.comment-branch.outputs.base_ref }}
 
     steps:
+      - name: Disabled on forks
+        if: ${{ github.event.pull_request.head.repo.full_name != github.repository }}
```

```suggestion
        if: ${{ github.event.pull_request.head.repo.fork }}
```

:nerd_face: 

(There is no functional difference though.)